src/client/pages/Offer/Checkout/Sign.tsx

```diff
@@ -77,6 +77,11 @@ export const Sign: React.FC<Props> = ({
       ? textKeys.CHECKOUT_SIMPLE_SIGN_BUTTON_TEXT()
       : textKeys.CHECKOUT_SIGN_BUTTON_TEXT()
 
+  const hasSignStatusSection =
+    checkoutMethod &&
+    signUiState !== 'NOT_STARTED' &&
+    !(signUiState === 'STARTED' && checkoutMethod === CheckoutMethod.SimpleSign)
```

Comment:
It's rather that I don't want to render the sign status section when there's no text in it (because it looks weird) but this was not the best solution for that. I think we have a better solution in the old `Sign` component so maybe we could pick things from there. 🙃